### DISCUSSION TITLE
JSON parse error

### DISCUSSION THREAD

**Discussion by neo269** [Athou/commafeed#1335] [General]
Using FeedMe to access Commafeed
used all the credentials to log in the App
When Synced I get "JSON parse error"
I cannot see any unread feeds in the app.

May you pls help
Thank you

**Comment by Athou** [comment]
AFAIK Feedbin is a platform similar to CommaFeed. What do you mean exactly by "Using Feedbin to access Commafeed"?

**Reply by neo269** [reply]
I m so sorry
I meant FeedMe Android app. My bad!

I put in all credentials into the app but it gives me the JSON parse error.

May you pls help me where am i going wrong

Thanks

**Reply by Athou** [reply]
You need to go to the Settings page of CommaFeed, find the "Fever API" section and read the instructions there.

**Reply by neo269** [reply]
I did that
Host https://www.commafeed.com/rest/fever/user/XXXXXXXX
username: my_username
password: API key

still the feeds are not getting synced giving the error

I tried with another RSS reader app called FocusReader which also had same issue!

I dont know where I am going wrong!

Thanks

Edit: Atatching Notification Error
<img width="136" alt="Screenshot_1" src="https://github.com/Athou/commafeed/assets/17139294/92fa2218-3f4d-4be1-a680-861ab2ccc6c7">

**Reply by Athou** [reply]
> Host https://www.commafeed.com/rest/fever/user/XXXXXXXX
> username: my_username
> password: API key

That should work, I don't know why it doesn't. I'm unfortunately not able to reproduce the issue :/

**Reply by neo269** [reply]
Thats strange :(

**Reply by Athou** [reply]
You could try to open an issue on the FeedMe repository : https://github.com/seazon/FeedMe/issues

**Reply by neo269** [reply]
Thanks
I have raised the issue @ Feedme repository

However I do face issue with Commafeed on FocusReader android app also
it says "error downloading new articles"

Thanks

**Reply by Athou** [reply]
Well actually, FeedMe and FocusReader both work with my local environment and with the test environment of commafeed, but not with commafeed.com

FeedMe is able to retrieve subscriptions and categories but not articles. I don't have an error, the entry list is just empty.
FocusReader shows the "error downloading new articles" message.

The problem may be that the entry ids on commafeed.com are much larger (e.g. 34220073284). It may be that the internal database of those apps are unable to store such large numbers.

**Reply by neo269** [reply]
Ok. Got it. Thank You

**Comment by kanyi** [comment]
I created an issue for this in the FeedMe GitHub site: https://github.com/seazon/FeedMe/issues/53
